administrator/components/com_joomgallery/models/control.php

```diff
@@ -185,4 +185,72 @@ protected function populateState()
 
     parent::populateState();
   }
+
+  /**
+   * Method to count the images and categories.
+   *
+   *
+   * @return  void
+   * @since   3.4.0
+   */
+  public function statisticinfo()
```

Comment:
Code style: I would name this function get**S**tatistics() or get**S**tatistic**I**nfo() following getImages() or getComments()